src/client/src/models/Scene.js

```diff
@@ -254,10 +253,20 @@ const Scene = {
     }
     return true
   },
-
+  
+  fileName: null,
+  
   getFileName: function() {
     return JSON.parse(localStorage.getItem('file-name'))
   },
+  
+  setFileName: function() {
```

Comment:
I don't necessarily have a problem with this, but it did make me aware of an issue with our code. It is kind of misleading given the other "set" functions. Most of the other "set" functions save to localStorage, with the exception of the "currentScene" one since it was useful to have direct manipulation of that variable.

I'm wondering if we should have a more consistent way of doing setting/saving things.

Is there a reason why you didn't just have setFileName save to localStorage? Maybe I need to look more closely, but it seems like you could have just had a get/set that just get and set localStorage, and then skipped having a separate save function.

Reply:
We update the local variable on change when using text boxes that way the display updates on change. That is your original design of this that I am replicating. The set function sets the local variable to what the local storage is, and the save variable sets the local storage to what the value of the local variable is. This is the uniform way we deal with text boxes right now, and I'm not inclined to change them when there are other things that need our attention.